src/RequestBuilder/ItemPropertiesModifierRequestBuilder.php

```diff
@@ -7,10 +7,8 @@
 use Lmc\Matej\Model\Command\ItemPropertySetup;
 use Lmc\Matej\Model\Request;
 
-class ItemPropertiesSetupRequestBuilder extends AbstractRequestBuilder
+class ItemPropertiesModifierRequestBuilder extends ItemPropertiesGetterRequestBuilder
```

Comment:
My preference is to keep this name as is - its meaning is already IMHO clear and distinctive from the GET version, and this adds no extra value - maybe even is confusing, because you actually cannot *modify* (like change data type) item properties once they are created in the database. So the the `ItemPropertiesSetupRequestBuilder` is for me ok. What do you think?